observability/otel/otel-configuring-otelcol-metrics.adoc

```diff
@@ -4,11 +4,21 @@
 include::_attributes/common-attributes.adoc[]
 :context: otel-configuring-metrics
 
-//[id="setting-up-monitoring-for-otel"]
-//== Setting up monitoring for the {OTELShortName}
-//The {OTELOperator} supports monitoring and alerting of each OpenTelemtry Collector instance and exposes upgrade and operational metrics about the Operator itself.
+The OpenTelemetry Collector provides its own internal metrics to monitor its performance and health. These metrics are exposed through a dedicated service created by the OpenTelemetry Operator.
 
-You can enable metrics and alerts of OpenTelemetry Collector instances.
+The operator automatically creates a service named `<instance name>-collector-monitoring` that exposes the internal metrics of the OpenTelemetry Collector. This service listens on port `8888` by default.
+
+These metrics can be used to monitor the Collector's performance, resource consumption, and other internal behaviors. You can use a Prometheus instance or another monitoring tool to scrape the metrics from the `-collector-monitoring` service.
+
+The metrics exposed include, but are not limited to:
+
+* Collector memory usage
+* CPU utilization
+* Number of active traces and spans processed
+* Dropped spans, logs or metrics
+* Exporter and receiver statistics
+
+You can create automatically the `ServiceMonitors` or `PodMonitors`` from the operator.
```

Comment:
Here, I think you could reuse my suggestion from one of the other files in this PR:
```suggestion
[NOTE]
====
When the `spec.observability.metrics.enableMetrics` field in the `OpenTelemetryCollector` custom resource (CR) is set to `true`, the `OpenTelemetryCollector` CR automatically creates a Prometheus `ServiceMonitor` or `PodMonitor` CR to enable Prometheus to scrape your metrics.
====
```